### PR TITLE
Ask for clarification of JIRA usage.

### PR DESCRIPTION
**Pull request by ktf**

### CODE REVIEW

**Comment by ktf** [comment]
@Barthelemy @dberzano I know is written somewhere, could we have a link to a JIRA quickstart or something like that?

**Comment by Barthelemy** [comment]
I will write something. I will call you to clarify the need.

**Comment by Barthelemy** [comment]
ok, I have just seen that you ahve actually put some code in there. I will put a sentence in the doc.

**Comment by ktf** [comment]
In principle you should be able to edit directly the file from this PR. Is that not the case?

**Comment by Barthelemy** [comment]
oh, ok, I will then.

File: README.md [hunk 1]
```diff
@@ -23,7 +23,8 @@ A quickstart page can be found under [https://aliceo2group.github.io/](https://a
 In order to build and install O2 with aliBuild you can follow [this tutorial](http://alisw.github.io/alibuild/o2-tutorial.html).
 
 ### Issue tracking system
-We use JIRA to track issues. [Report a bug here](https://alice.its.cern.ch/jira/secure/CreateIssue.jspa?pid=11201&issuetype=1).
+We use JIRA to track issues. [Report a bug here](https://alice.its.cern.ch/jira/secure/CreateIssue.jspa?pid=11201&issuetype=1). 
+Add the JIRA issue key (e.g. `O2-XYZ`) to the PR title or in a commit message to have the PR/commit appear in the JIRA ticket.
```
Comment (matthiasrichter) [inline README.md]: nice! didn't know that.

Comment (ktf) [reply README.md]: yes, it's nice, albeit the link is a bit hidden. We should also have some bot which goes around the issues and substitutes `O2-XYZ` with an actual link to our JIRA.

Comment (ktf) [reply README.md]: Actually it looks like it's only available for the JIRA Cloud.

https://github.com/integrations/jira

Comment (Barthelemy) [reply README.md]: @matthiasrichter 👍 

@ktf yes, the bot exists but only for JIRA Cloud 😦   I have already told Atlassian what I think of it. 

So we have a one way integration that works well (GH->JIRA) and the other that is inexistant. As you mention, it is a simple bot to replace `O2-XYZ` into the full URL. It should not be complicated. Maybe a service task ?

Comment (ktf) [reply README.md]: Might as well, albeit I wouldn't say it's urgent.

**Pull request merged**